app/assets/javascripts/application.js

```diff
@@ -233,6 +237,7 @@ function template_info(div, url) {
     data: form,
     success: function(response, status, xhr) {
       $(div).html(response);
+      $('select').select2();
```

Comment:
can you generalize it to a separate function, so instead of after every ajax have a select statement,  you have a call to a function that execute after ajax response?